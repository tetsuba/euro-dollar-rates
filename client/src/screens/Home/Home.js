import React from 'react'
import { Link } from 'react-router-dom';

export default class Home extends React.Component {

  render() {
    return (
      <div className="row mt-3">
        <div className="card" style={{width: '18rem'}}>
          <div className="card-body">
            <h5 className="card-title">Market Watch</h5>
            <Link to="market-watch" className="btn btn-primary">Click here</Link>
          </div>
        </div>
      </div>
    )
  }
}
