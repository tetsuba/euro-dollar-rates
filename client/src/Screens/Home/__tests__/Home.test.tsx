import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import Home from '../Home'

describe('MarketWatch', () => {
    test('should render screen', () => {
        const { asFragment } = render(
            <MemoryRouter>
                <Home />
            </MemoryRouter>
        )
        expect(asFragment()).toMatchSnapshot()
    })
})
