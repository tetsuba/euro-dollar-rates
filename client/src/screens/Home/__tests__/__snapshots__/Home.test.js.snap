// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarketWatch should render screen 1`] = `
<DocumentFragment>
  <div
    class="row mt-3"
  >
    <div
      class="card"
      style="width: 18rem;"
    >
      <div
        class="card-body"
      >
        <h5
          class="card-title"
        >
          Market Watch
        </h5>
        <a
          class="btn btn-primary"
          href="/market-watch"
        >
          Click here...
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
